models/xhani2023xmlpo/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+title: Explainable Machine Learning Pipeline Ontology
+acronym: XMLPO
+issued: 2023
+modified:
+contributor:
+ - https://orcid.org/0000-0003-2536-0574
+ - https://orcid.org/0000-0002-4547-7000
+keyword:
+ - explainability
+ - machine learning models
+theme: Class T - Technology
+editorialNote:
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+context:
+ - research
+source:
+ - https://doi.org/10.3990/1.9789036558594
```

Comment:
This link is broken

Reply:
1) The ontology.ttl file was included.
2) The link was provided by the author and, as far as I know, it is correct. It is the permanent identifier of the thesis. It does not redirect to the thesis because of publication restrictions.

Reply:
Got it.